hours.js

```diff
@@ -38,7 +38,13 @@ function Slot(date, start, stop) {
 // check if a date is a workday
 function isWorkday(date) {
     var dn = date.day();
-    return dn < 6 && dn > 0;
+    var notsunday =  dn < 6 && dn > 0;
+
+    var datestr = date.format("ddMMYY");
```

Comment:
I have not tested this, but holidays seems to store dates in "YYYYMMdd" format. Should this not also be the case here?